src/families.h

```diff
@@ -93,6 +96,11 @@ class Gaussian : public Family {
     }
   };
 
+  double Deviance(const arma::mat& prediction,
+                  const arma::mat& y) {
+    return 1.0 -
+      arma::accu(arma::square(prediction - y))/arma::accu(arma::square(y));
+  }
```

Comment:
Maybe call this "fraction of deviance" or something (or at least add a comment to that effect). The `/arma::accu(arma::square(y))` part makes it not a "real deviance".